doc-src/concepts/bc.rst

```diff
@@ -0,0 +1,180 @@
+fully reflected back, analogous to a 1D transmission line terminated
+by a short circuit.
+
+**Perfect Magnetic Conductor (PMC)**: By duality, it sets the magnetic
+field to 0 at the boundary. It acts as a boundary at which
+the magnetic field lines can't penetrate. It's equivalent to a Perfect
+Magnetic Conductor (PMC) with infinite permeability, also known as the
+Magnetic Wall. All incoming waves are fully reflected back as well,
+but with a phase opposite to that of the PEC, analogous to a 1D
+transmission line terminated by an open circuit. It's used mainly as
+a mathematical tool to enforce field symmetry when simulating a
+half-structure, as no natural material in the real world behaves like
+a PMC.
+
+Mathematically, both PEC and PMC are Dirichlet boundary conditions
+that enforce fixed field values (e.g. zero).
+
+.. note::
+   For our simulation, instead of explicitly modeling metal
+   plates, we can model a vacuum with nothing inside, taking advantage
+   of the PEC boundary conditions at ``z_min`` and ``z_max`` for
+   fast computation. However, this is out of this tutorial's scope,
+   and it won't capture the fringe fields above and below. We won't use
+   PEC or PMC in this example.
+
+Absorbing Boundary Conditions (ABC)
+-------------------------------------
+
+In open-boundary problems, such as antennas or structures with radiation
+loss, *Absorbing Boundary Conditions (ABC)* must be used to suppress
+reflections to prevent spurious simulation results. At its boundaries, the
+electromagnetic waves are absorbed and dissipated without reflecting back,
+creating the illusion of an infinitely large free space. If PEC and PMC
+are analogous to a shorted or opened transmission line, the ABC is analogous
+to a termination resistor.
+
+In general, these simulations should use a simulation box larger than the
+structure to avoid intrusion of strong fringe fields at the boundary.
+However, deliberately running a transmission line into the PML (defined
+below) can be used as a perfect termination. This allows one
+to terminate a transmission line without knowing its characteristic
+impedance, enabling impedance measurement. In fact, openEMS's microstrip
+ports don't create a lumped termination resistor by default, expecting
+users to place them into the PML at boundaries.
+
+.. figure:: images/anechoic_chamber.jpg
+   :class: with-border
+   :width: 60%
+
+   A simulation box with PML at all boundaries acts like an anechoic chamber
+   in an EMC test lab, mimicking an infinitely large free space. Image by
+   Adamantios (at English Wikipedia), licensed under CC BY-SA 3.0.
+
+Two kinds of Absorbing Boundary Conditions are implemented in openEMS.
+
+#. **Mur's Boundary Condition (MUR)**. This is a
+   first-generation boundary condition purely defined by differential
+   equations, originally invented by Gerrit Mur in the 1980s. It has
+   a moderate computational overhead, but it works only if the EM wave
```

Comment:
it works only at a orthogonal direction is not really correct. It works best (even perfect) for orthogonal direction, but it still works quite decent for other angles too. It just gets worse and worse the further away from orthogonal you get...